plexupdate.sh

```diff
@@ -493,6 +493,9 @@ if [ "${AUTOINSTALL}" = "yes" ]; then
 	if [ ${RET} -ne 0 ]; then
 		# Clarify why this failed, so user won't be left in the dark
 		error "Failed to install update. Command '${DISTRO_INSTALL} "${DOWNLOADDIR}/${FILENAME}"' returned error code ${RET}"
+		if [ "${DISTRO}" = "synology" -a ${RET} -eq 1 ]; then
+			echo -e "On Synology devices, you need to add Plex's public key to Package Center. If you have not done so, follow the instructions at \033[0;34mhttps://support.plex.tv/articles/205165858-how-to-add-plex-s-package-signing-public-key-to-synology-nas-package-center/\033[0m"
```

Comment:
Since `synopkg` seems to return an exit code of 1 for any and all errors, I'd add in a check for `error = [289]` in the output. (returned on STDOUT even though it's an error, and with an exit code of 1 even though it's obviously 289... way to go Synology... 🤦‍♂)

Reply:
Actually, never mind. We'd have to wrap `DISTRO_INSTALL` with logic to capture the output, and then we'd have to monitor both STDOUT and STDERR and preferably split them out when re-printing them again which would just be a mess.

Though I would remove the `-e` and the color codes. Since this error could theoretically get triggered while running as a scheduled task, color codes could end up garbling the output, so it's best to stick to plain text I think. (and that blue color shows up nearly unreadable on my terminal)